test/integration/ae_mdw_web/controllers/tx_controller_test.exs

```diff
@@ -164,78 +161,45 @@ defmodule Integration.AeMdwWeb.TxControllerTest do
   describe "txs_direction only with direction" do
     test "get transactions when direction=forward", %{conn: conn} do
       limit = 33
-      conn = get(conn, "/txs/forward?limit=#{limit}")
-      response = json_response(conn, 200)
 
-      assert Enum.count(response["data"]) == limit
+      assert %{"data" => txs, "next" => next} =
+               conn |> get("/txs/forward", limit: limit) |> json_response(200)
 
-      Enum.each(response["data"], fn data ->
-        assert data["tx_index"] in 0..(limit - 1)
-      end)
-
-      {:ok, data, _has_cont?} =
-        Continuation.response_data({TxController, :txs, %{}, build_scope(conn.assigns), 0}, limit)
+      txis = Enum.map(txs, fn %{"tx_index" => tx_index} -> tx_index end)
 
-      assert ^data = response["data"]
+      assert ^txis = Enum.to_list(0..(limit - 1))
```

Comment:
Could we add an assertion for the first hashes too? It would helpful to detect the sync is performed equally across multiple resyncs.

Reply:
For the first 99 txs maybe (as there was problem once with one txi around 80) and I wouldn't say for the last 99 because they change but assert the same tx hash for the txs between 500_000 and 500_100.

Reply:
These are additional changes that won't undo the changes introduced in this PR. Perhaps we can create a separate issue to add what you're asking for?

Reply:
Sure #433